### PR TITLE
display proxy configuration 

### PR DESCRIPTION
**Pull request by crazy-max**
related to https://github.com/docker/build-push-action/issues/822
follow-up https://github.com/docker/actions-toolkit/pull/114

image

### CODE REVIEW

File: test/proxy.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# syntax=docker/dockerfile:1
+FROM alpine
+RUN apk add --no-cache curl net-tools
+ARG HTTP_PROXY
+ARG HTTPS_PROXY
+RUN printenv HTTP_PROXY
+RUN printenv HTTPS_PROXY
```
Comment (thaJeztah) [inline test/proxy.Dockerfile:7]: Just really glancing over, so no idea if relevant at all;

- both `http_proxy` (lowercase) and `HTTP_PROXY` are a thing (some software use lower, other uppercase)
- do we need some verification for `NO_PROXY` / `no_proxy` as well?

Comment (crazy-max) [reply test/proxy.Dockerfile:7]: Both work yes: https://docs.docker.com/engine/reference/builder/#predefined-args

Comment (crazy-max) [reply test/proxy.Dockerfile:7]: Oh you mean reading the var in the Dockerfile, let me check

Comment (crazy-max) [reply test/proxy.Dockerfile:7]: Yes we set (upper|lower)case proxy envs: https://github.com/moby/buildkit/blob/cc2438f7ebd3fdeec4c4cb50d59ecdd6d3d9dcc5/solver/llbsolver/ops/exec.go#L383-L401

**Review by tonistiigi — APPROVED** [review]

**Pull request merged**